prac_08/dynamic_labels.kv

```diff
@@ -0,0 +1,14 @@
+BoxLayout:
+    BoxLayout:
+        orientation: 'vertical'
+        id: main_layout
+
+    BoxLayout:
+        orientation: 'horizontal'
+        size_hint_y: 0.2
+        Button:
+            text: "Create Labels"
+            on_press: app.create_labels()
+        Button:
+            text: "Clear Labels"
+            on_press: app.clear_all_labels()
```

Comment:
I think in this program, we are supposed to not have any buttons so, the button section is changed to label and there is not supposed to have any user interaction or input